src/camera.py

```diff
@@ -42,18 +46,19 @@ class Camera:
     MIN_TILT_CALI_DEGREE = -150
     """The minimum value to calibrate tilt."""
 
-    def __init__(self, bus_number=1, pan_channel = 1, tilt_channel = 2):
-        picar.setup()
-
+    def __init__(self, bus_number=1, pan_channel = 1, tilt_channel = 2,
```

Comment:
I'd rather add module level constants

```python
RELATIVE = 0
ABSOLUTE = 1
```
and then name the parameter `pan_type`

Reply:
and then you can default to absolute if you want.

```python
pan_type = ABSOLUTE
```

Reply:
And then if a value other than 0 or 1 is passed in, default to absolute